win/WIN_ButtonGroup.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <vector>
+#include <memory>
+
+namespace win
+{
+
+	class ToggleButton;
+	
+	class ButtonGroup
+	{
+	public:
+		ButtonGroup();
+		~ButtonGroup() = default;
+		ButtonGroup(const ButtonGroup& that) = default;
+		ButtonGroup(ButtonGroup&& that) = default;
+		ButtonGroup& operator=(const ButtonGroup& that) = default;
+		ButtonGroup& operator=(ButtonGroup&& that) = default;
+
+		void addButtonChild(std::shared_ptr<ToggleButton> const& child);
+		std::vector<std::shared_ptr<ToggleButton>> getButtonChildren() const { return buttonChildren_; }
```

Comment:
Return a reference to avoid copying the collection?
**SHOULD**

Reply:
done